examples/k8s/deploy/linstor-csi-1.17.yaml

```diff
@@ -49,7 +49,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-snapshotter
-          image: quay.io/k8scsi/csi-snapshotter:v1.2.2
+          image: quay.io/k8scsi/csi-snapshotter:v2.0.1
           args:
             - "--csi-address=$(ADDRESS)"
             - "--timeout=120s"
```

Comment:
@w00jay, like just put `--timeout=4m` here?
Excuse me. I little confused, shouldn't we also update timeout option for other images as well?
BTW, you can always do that in separated commit

Reply:
It seems the parameter's changed only for the snapshotter for the moment.  I will follow up.  Thank you!